src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs

```diff
@@ -2,4074 +2,938 @@
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
This file was `Vector_Operations.cs` + some trailing bits from `Vector.cs`, it now just contains `static class Vector`.

Reply:
This file is largely untouched as it, minus the Widen/Narrow and Same-Size Conversion functions were already not templated. Just the `#regions` were removed.